metafacture-io/src/test/java/org/metafacture/io/ObjectFileWriterTest.java

```diff
@@ -105,6 +104,14 @@ public void shouldIncrementCountOnResetBeforeStartingNewFile() throws IOExceptio
         assertTrue(new File(tempFolder.getRoot(), "test-1").exists());
     }
 
+    @Test
+    public void issue543_shouldResultEmptyWhenNothingIsProcessed() throws IOException {
+        writer.process("");
+        writer.closeStream();
+
+        assertOutput( "");
```

Comment:
Extra whitespace.

Reply:
Thx. Addressed in d3a47eeb70166d1ffe66ffd1cf7f3926dbc89c4d.